xpdacq/beamtime.py

```diff
@@ -112,28 +112,44 @@ def _check_mini_expo(exposure, acq_time):
                                  " to 0.5s"))
 
 
-def ct(dets, exposure, *, md=None):
+def _shutter_step(detectors, motor, step):
+    """ customized step to ensure shutter is open before
+    reading at each motor point and close shutter after reading
     """
-    Take one reading from area detectors with given exposure time
+    yield from bp.checkpoint()
+    yield from bp.abs_set(motor, step, wait=True)
+    yield from bp.abs_set(xpd_configuration['shutter'],
+                          XPD_SHUTTER_CONF['open'], wait=True)
+    yield from bp.trigger_and_read(list(detectors) + [motor])
+    yield from bp.abs_set(xpd_configuration['shutter'],
+                          XPD_SHUTTER_CONF['close'], wait=True)
+
+
+def ct(dets, exposure):
+    """
+    Take one reading from area detector with given exposure time
 
     Parameters
     ----------
     dets : list
-        list of 'readable' objects
+        list of 'readable' objects. default to area detector
+        linked to xpdAcq.
     exposure : float
         total time of exposrue in seconds
-    md : dict, optional
-        extra metadata
 
-    Note
-    ----
-    area detector that is triggered will always be the one configured in
-    global state. Please refer to http://xpdacq.github.io for more information
+    Notes
+    -----
+    area detector being triggered will  always be the one configured
+    in global state. To find out which these are, please using
+    following commands:
+
+        >>> xpd_configuration['area_det']
```

Comment:
I like this improved documentation.  Thanks so much for doing these details, it really makes a difference.